services/core-api/tests/helpers.py

```diff
@@ -0,0 +1,7 @@
+from datetime import datetime
+
+def get_datetime_iso8601_string(date: datetime):
```

Comment:
added some datetime testing helpers in a generic "helpers" file. Our version of python doesn't very easily generate the ISO 8601 string, but I thought this wasn't _too_ clunky.